Resources/Locale/en-US/foldable/components/foldable-component.ftl

```diff
@@ -11,3 +11,8 @@ fold-flip-verb = Flip
 
 fold-zip-verb = Zip up
 fold-unzip-verb = Unzip
+
+# Moffstation - Begin (Toggleable Security Helmet)
+fold-attach-visor-verb = Attach Visor
+fold-detach-visor-verb = Detach Visor
+# Moffstation - End
```

Comment:
I'd say just throw these in a new `ftl` file under a `_Moffstation` directory somewhere.